csrc/executor.cpp

```diff
@@ -1891,4 +1893,267 @@ float FusionExecutor::runRtc(
   return kernel_time_ms;
 }
 
+flatbuffers::Offset<serde::FusionExecutor> FusionExecutor::serialize(
```

Comment:
I'm debating whether we want to have serialization/deserialization logic here in the executor.cpp file. Does a dedicated serde helper make sense at all? something like `executor_serde.cpp`.

Just trying to separate the logic out to keep things easier to read.

Reply:
but that does give us lots of files with serde suffix... maybe a single place where we keep all of it?!

Reply:
`FusionExecutor` has several private variables and nested classes, so it was easier to have the `serde` functions be member functions.

Reply:
I've been creating separate logic if all information is specified by the constructor. e.g., `ArgAbstract` classes